src/_content/posts/introducing-keynote-natalia-bidart.md

```diff
@@ -0,0 +1,52 @@
+---
+author: DjangoCon US Communications Team
+category: General
+published_datetime: 2024-08-13 06:00:00-04:00
+cover:
+    url: /assets/img/blog/natalia_bidart.jpeg
+    alt: "Natalia Bidart"
+layout: post
+title: "Announcing Our Keynotes: Natalia Bidart"
+---
+
+### We're honored to announce our third keynote speaker for DjangoCon US 2024 and Django Fellow: Natalia Bidart
+
+We had the opportunity to sit down with Natalia and ask a few questions about her work, upcoming talk, and what she is looking forward to at DjangoCon US 2024.
+
+## Can you tell us how you use Django is our go to question, but how has your time as a Django Fellow been so far?
+
+>I joined April last year. So it's been a year and four months, that I've been a fellow, and it has been great, very challenging in that prior to to being a fellow, I wasn't that involved in contributing to Django, so there was an initial period when it was everything was new to me, and I mean everything like how to do things, Reading the docs, when it comes to contributing, building PRs, reviewing PRs, triage and tickets and getting used to formats and guidelines for the code, the commit messages, the bulk triage flows in different stages. So, it was definitely a non trivial period and non trivial adaptation to get used to. At times I felt a little bit inadequate for the position, so I tried to shift my focus to getting to know the community and the different ways of doing things, I tried to put my energy to where I thought it could provide the most value, which was around engaging with the community and trying to ease different interactions and attend to those prs. So, yes it has been a very challenging process, but I have also learned a lot and I am honored to be a Django fellow.
+
+## Can you give us a teaser about what your talk is about?
+
+>I'd like to talk about the Django Fellowship program, which I'm a part of. This program is crucial for maintaining Django’s current status as a secure, robust, and progressive framework. One of the key reasons to contribute to the Django Software Foundation (DSF) is that a significant portion of donations goes directly to funding this fellowship.
```

Comment:
```suggestion
>I'd like to talk about the Django Fellowship program, which I'm a part of. This program is crucial for maintaining Django’s current status as a secure, robust, and progressive framework. One of the key reasons to contribute to the Django Software Foundation (DSF) is that a significant portion of donations go directly to funding this fellowship.
```

Reply:
Goes is correct here. It has to agree with "portion", not "donations" (which is inside the prepositional phrase)